gem_tasks/cucumber.rake

```diff
@@ -3,25 +3,8 @@
 require 'cucumber/rake/task'
 require 'cucumber/platform'
 
-class Cucumber::Rake::Task
```

Comment:
I've read the commit message before writing the comment. I've read it again and still don't see how this helps to reproduce the problem.

How did you reproduce it? By running Prezento's acceptance tests or by running kolekti_metricfu's acceptance tests? What happens if you run cucumber using an undefined profile?